prime-router/metadata/HL7/catchall/hl7/datatypes/XCN/RelatedPerson.yml

```diff
@@ -0,0 +1,190 @@
+# $schema: ./../../../../../json_schema/fhir/hl7-to-fhir-mapping-resource-template.json
+
+resourceType: RelatedPerson
```

Comment:
The XCN[RelatedPerson] mapping needed to be implemented to support the PRT segment. There are differences from XCN[Practitioner] (HL7 > FHIR) but the two are able to share their FHIR > HL7 mappings.